server/index.js

```diff
@@ -7,17 +7,18 @@ const port = 8000;
 const db = require('../database/config.js')
 
 app.use(express.static(path.join(__dirname, '../public')));
+app.use(parser.json());
 
 app.get('/media', (req, res) => {
-  // console.log('made it to endpoint!');
+  const game = req.body;
```

Comment:
should destructure game because all hail airbnb
const { game } = req.body;

Reply:
Talked about it with you via slack! going to merge for now but discuss more thoroughly with the group tonight during recap!